### PR TITLE
Adds a fixture to generate random test values for channel settle. 

### PR DESCRIPTION
**Pull request by err508**

### CODE REVIEW

**Comment by loredanacirstea** [comment]
Please add a proper title for what the PR is and a short description. Also, please put the issue link in the description, because it's a pain to search for the issue instead of just clicking a link.

**Comment by loredanacirstea** [comment]
Also, branch the PR from `master`, to not include already reviewed code in the diff. This branch has nothing to share with the branch from https://github.com/raiden-network/raiden-contracts/pull/104

**Comment by err508** [comment]
Related to Issue #89

File: raiden_contracts/tests/fixtures/channel_test_values_random.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                )
+                for withdrawn, transferred, deposit, locked in
+                [sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]
+            ]),
+            False
+        )
+    ]
+
+
+channel_settle_test_values_random = channel_settle_test_values_random(100)
```
Comment (loredanacirstea) [inline raiden_contracts/tests/fixtures/channel_test_values_random.py]: Where is this used? Is there a test using this?

File: raiden_contracts/tests/fixtures/channel_test_values_random.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        (
+            tuple([
+                ChannelValues(
+                    deposit=deposit,
+                    withdrawn=withdrawn,
+                    transferred=transferred,
+                    locked=locked
+                )
+                for withdrawn, transferred, deposit, locked in
+                [sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]
```
Comment (loredanacirstea) [inline raiden_contracts/tests/fixtures/channel_test_values_random.py]: As mentioned, it is hard to understand what you are trying to achieve here. Either use readable code or make sure to comment. The two cases look similar and you have to pay attention to see that the `withdrawn, transferred, deposit, locked` values are actually reordered (though I don't really understand why).

`[sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]` seems like you are choosing sets of 4 sorted random values. In the `True` case, you will always have `deposit <= withdrawn <= transferred <= locked`, which is not really useful and not correct. Am I wrong?

**Pull request closed without merging**